pkg/helm/charts.go

```diff
@@ -59,6 +59,40 @@ func (v InstalledReleases) Less(i, j int) bool {
 	return v[i].Version < v[j].Version
 }
 
+func GetChartSecret(releaseName, namespace, version string) (*helmrelease.Release, error) {
+	clientSet, err := k8sutil.GetClientset()
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to get clientset")
+	}
+
+	selectorLabels := map[string]string{
+		"owner":   "helm",
+		"name":    releaseName,
+		"version": version,
+	}
+	listOpts := metav1.ListOptions{
+		LabelSelector: labels.SelectorFromSet(selectorLabels).String(),
+	}
+
+	secrets, err := clientSet.CoreV1().Secrets(namespace).List(context.TODO(), listOpts)
+	if err != nil {
+		if kuberneteserrors.IsNotFound(err) {
+			return &helmrelease.Release{}, nil
```

Comment:
`return nil, nil` or just remove this check.  the caller should check that the app is not found.

The problem is that `Release` contains pointers that will be nil here and will cause panics.  `Info` is a pointer and `releaseSecret.Info.Description` will panic.

Reply:
done, returning nil, nil